badge_server/main.py

```diff
@@ -12,59 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""
-URL for creating the badge:
-[TODO] Switch to use pybadges once figure out the module not found issue.
-'https://img.shields.io/badge/{name}-{status}-{color}.svg'
-"""
-import datetime
-import logging
-import requests
-import threading
-
+"""A HTTP server for badges"""
 
+import datetime
 import flask
 import pybadges
 
 import utils as badge_utils
+from compatibility_lib import utils as compat_utils
```

Comment:
> Use from x import y as z if two modules named y are to be imported

That is the case here with the local (badge server) `utils` and compatibility lib `utils` both being imported into the same file and I think it would be confusing to have one be more specific than the other.

Reply:
Got it, thanks!